projects/iio_demo/src/platform/xilinx/parameters.h

```diff
@@ -42,26 +43,12 @@
 /******************************************************************************/
 /***************************** Include Files **********************************/
 /******************************************************************************/
-
-#ifdef XILINX_PLATFORM
 #include <xparameters.h>
-#endif
-
-#ifdef ADUCM_PLATFORM
-#include "irq_extra.h"
-#endif
-
-#ifdef STM32_PLATFORM
-#include "stm32_hal.h"
-#endif
+#include <xil_cache.h>
 
 /******************************************************************************/
 /********************** Macros and Constants Definitions **********************/
 /******************************************************************************/
-
-
-#ifdef XILINX_PLATFORM
-
 #ifdef _XPARAMETERS_PS_H_
 #define ADC_DDR_BASEADDR	(XPAR_DDR_MEM_BASEADDR + 0x800000)
 #define DAC_DDR_BASEADDR	(XPAR_DDR_MEM_BASEADDR + 0xA000000)
```

Comment:
Let's allocate statically buffers here, the same as the other platforms. I know it's not exactly related as a change, but it's a long time needed change.